src/table/utils/handle-key-press.ts

```diff
@@ -301,6 +307,9 @@ export const handleBodyKeyDown = ({
     case 'Tab':
       focusSelectionToolbar(evt.target as HTMLElement, keyboard, evt.shiftKey);
       break;
+    case 'c':
+      (evt.ctrlKey || evt.metaKey) && handleKeyToCopyCell(cell);
```

Comment:
We need a feature flag here before this is merged. I will find out what it should be